### ISSUE TITLE
Getting an installation failure

### ISSUE THREAD

**Issue by decapo01** [digital-asset/ghcide#220]
Following the install instructions

```
First install the ghcide binary using stack or cabal, e.g.

1. git clone https://github.com/digital-asset/ghcide.git
2. cd ghcide
3. cabal install or stack install (and make sure ~/.local/bin is on your $PATH)
```

and I'm getting an error installing

```
haskell-lsp> Registering library for haskell-lsp-0.18.0.0..
                         
--  While building package hie-bios-0.2.1 using:
      /home/decapo01/.stack/setup-exe-cache/x86_64-linux/Cabal-simple_mPHDZzAJ_2.4.0.1_ghc-8.6.5 --builddir=.stack-work/dist/x86_64-linux/Cabal-2.4.0.1 build --ghc-options " -fdiagnostics-color=always"
    Process exited with code: ExitFailure 1
Progress 2/3
```

Let me know if I can give more info to help

**Comment by cocreature** [comment]
That looks like an error while building `hie-bios` that is coming from stack so I don’t think there is much we can do to fix this in `ghcide`. I would try to first run `stack clean --full` and if that still doesn’t help, maybe remove `~/.stack`.

**Comment by decapo01** [comment]
I'm not 100% sure if this fixed it but I am using ubuntu 18.04 and came across this post 
https://github.com/commercialhaskell/stack/issues/1012

I didn't follow the advice because I didn't have a `libtinfo.so` when I did a locate.

So what I did was install the libtinfo-dev package (`sudo apt install libtinfo-dev`) and it still didn't show up when I ran the `locate libtinfo.so` command, but i just took a shot and did `stack install` and it worked!

**Comment by cocreature** [comment]
Glad you managed to get it working! That looks like a stack issue rather than a `ghcide` issue so I’ll close this.
